test/custom-field-slot-test.html

```diff
@@ -29,6 +29,26 @@
     </script>
   </dom-module>
 
+  <dom-module id="x-field2">
+    <template>
+      <vaadin-custom-field id="customField">
+        <div id="wrapper">
+          <slot></slot>
+        </div>
```

Comment:
Same as current `<x-field>` but additionally has this wrapper element as a parent for the slot.